### ISSUE TITLE
Can't create new custom_field_group

### ISSUE THREAD

**Issue by max2320** [owen2345/camaleon-cms#666]
camaleon_cms (2.4.3.8) app/models/camaleon_cms/custom_field_group.rb

undefined method `map' for #<ActionController::Parameters:0x007f121685ad98> Did you mean? tap
```
  def add_fields(items, item_options)

    self.fields.where.not(id: items.map { |_k, obj| obj['id'] }.uniq).destroy_all
```

```
>> items
=> <ActionController::Parameters {"_1499451379_18323"=><ActionController::Parameters {"id"=>"", "name"=>"Untitled Text Box-0", "slug"=>"untitled-text-box-0", "description"=>""} permitted: true>} permitted: true>
>> items.map
!! #<NoMethodError: undefined method `map' for #<ActionController::Parameters:0x007f121685ad98>
Did you mean?  tap>
```

**Comment by haffla** [comment]
I think a little bit more information on what you are trying to do would be helpful. Can you show your code that is calling `add_fields`?

**Comment by max2320** [comment]
I fixed this in #667, the issue was caused by a Rails compatibility
